plugins/org.python.pydev.refactoring/src/org/python/pydev/refactoring/ast/visitors/position/IndentVisitor.java

```diff
@@ -90,7 +90,14 @@ public Object visitAssert(Assert node) throws Exception {
 
     @Override
     public Object visitAssign(Assign node) throws Exception {
-        visit(node.targets[0]);
+        if (node.targets.length > 0) {
+            visit(node.targets[0]);
+        }
+
+        if (node.type != null) {
```

Comment:
Actually, in this case we don't want to visit the `node.type`, just really the `node.targets[0]` (it's just computing the indent of the assign, so, visiting other nodes isn't necessary).

Reply:
Ok, so this will no longer need to check if node.target.length > 0, right?

Reply:
No, we still need that check, what we don't need is to visit the `node.type`.